src/js/GaModule.js

```diff
@@ -158,11 +158,12 @@ goog.require('ga_waitcursor_service');
     gaLayersProvider.wmtsMapProxyGetTileUrlTemplate =
         gaGlobalOptions.mapproxyUrl +
         '/1.0.0/{Layer}/default/{Time}/{TileMatrixSet}/{z}/{x}/{y}.{Format}';
-    gaLayersProvider.wmtsToDUrlTemplate = '//tod{s}.prod.bgdi.ch/' +
-        '1.0.0/{Layer}/default/{Time}/{TileMatrixSet}/{z}/{x}/{y}.{Format}';
-    gaLayersProvider.wmtsToD03UrlTemplate = '//tod{s}.prod.bgdi.ch/' +
-        '1.0.0/{Layer}/default/{Time}/{TileMatrixSet}/{z}/{y}/{x}.{Format}';
-    gaLayersProvider.dfltToDSubdomains = ['100', '101', '102', '103', '104'];
+    gaLayersProvider.wmtsToDUrlTemplate = gaGlobalOptions.wmtsUrl +
+        '/1.0.0/{Layer}/default/{Time}/{TileMatrixSet}/{z}/{x}/{y}.{Format}';
+    gaLayersProvider.wmtsToD03UrlTemplate = gaGlobalOptions.wmtsUrl +
+        '/1.0.0/{Layer}/default/{Time}/{TileMatrixSet}/{z}/{y}/{x}.{Format}';
+    gaLayersProvider.dfltToDSubdomains = gaGlobalOptions.staging === 'dev' ?
```

Comment:
If you remove the {s} from rc_dev you don't need to check the staging, subdomains are just ignored.